test/stressLinear.cxx

```diff
@@ -120,6 +120,11 @@
 #include "TMatrixDEigen.h"
 #include "TMatrixDSymEigen.h"
 
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wself-assign-overloaded"
```

Comment:
The diagnostic switch seems available only for clang master. Wouldn't that pragma be a problem for earlier versions of clang?